### ISSUE TITLE
a shortcut to boost the volume beyond 2x.

### ISSUE THREAD

**Issue by audiogamer22** [belaviyo/volume-booster#3]
I'm blind and can't shift plus click, my screen reader won't let me, it goes to other parts of the page, would it be possible to add a keyboard shortcut to do this? If it doesn't work with my screen reader I would be able to turn it off and after pressing hte shortcut, I can turn it back on again.

**Comment by audiogamer22** [comment]
You didn't reply to my thread and just marked it as complete, did you add a shortcut?

**Comment by belaviyo** [comment]
Sure, as of the next release, you can use "chrome://extensions/shortcuts" to assign shortcuts for `2x`, `3x`, and `4x` boosting. The shortcuts toggle the boosting level and adjust the new value same as Shift + Click.

**Comment by audiogamer22** [comment]
That's great! Will that work in firefox too? Or just chrome? I'm a firefox user.

**Comment by audiogamer22** [comment]
I think making the shortcuts something like, control 2 for 2x, control 3 for 3x and control 4, for 4x as a default would be really nice.

Could that be doable?

**Comment by belaviyo** [comment]
Both. After the next release on Firefox, go to "about:addons" and press the gear button. Then use "Manage Extension Shortcuts" to assign a custom shortcuts.

**Comment by audiogamer22** [comment]
That's great! Would you be able to asign shortcuts as a default so we'd not have to do that, honestly i'd probably forget it but if there were default shortcuts like, control, 2 3 and 4, for 2x 3x and 4x, I think that would be easier.

If you can't that's fine, I'd like to thank you for at least adding this feature.

Is there a way I can donate to you to say thank you? You honestly do deserve it.

**Comment by audiogamer22** [comment]
Oh also, do you have a timeline on that release by any chance?

**Comment by belaviyo** [comment]
> control 2 for 2x

The functionality varies depending on the operating system. On macOS, `Ctrl + 2` functions properly, but on Windows, I guess it switches browser to tab number two. I prefer not to set default values since it could confuse certain users and might not function across all operating systems.

**Comment by audiogamer22** [comment]
That's completely fair. could you please add a note somewhere in the addon to remind shortcut users how to asign new shortcuts? Thanks.

**Comment by belaviyo** [comment]
>  do you have a timeline on that release by any chance?

Firefox version is already released. Chrome is on review.

> could you please add a note somewhere in the addon to remind shortcut users how to asign new shortcuts? Thanks.

It is mentioned on the tooltip of the button. Also the FAQs page.

> Is there a way I can donate to you to say thank you

It was a useful feature for all users. If you want there is a link on the FAQs page.
https://webextension.org/listing/volume-booster.html?from=youtube

**Comment by audiogamer22** [comment]
I'm reading the FAQ page on the link you gave and there's no mention of shortcuts, do i have to wait for it to update?

**Comment by belaviyo** [comment]
Yes, it is cached; FAQ6

https://webextension.org/listing/volume-booster.html?from=youtube&rev

**Comment by audiogamer22** [comment]
I tried  to reply to the email in thunderbird and so far nothing's happened but you might see a similar reply. 

I'm new to github, what does cashed mean? also I have 0.1.3
Last Updated
16 January 2024. I've tried closeing firefox and it didn't update the addon even though I have auto update on, do I need to redownload the addon or is this normal?

**Comment by belaviyo** [comment]
Open "about:addons" in a browser tab and use the gear button, then "Check for Updates".

**Comment by audiogamer22** [comment]
Thanks so much!. This works great! I'll be donating about £20 when I can to say thanks. I know it's not needed but I do apreciate it.

**Comment by belaviyo** [comment]
Cheers

**Comment by audiogamer22** [comment]
I'm not sure if the request is closed now but if it isn't, feel free to close it.
